pkg/anonymiser/anonymiser.go

```diff
@@ -49,9 +49,10 @@ func (a *anonymiser) ReadTable(tableName string, rowChan chan<- *database.Row) e
 	// Anonimise the rows
 	go func() {
 		for {
-			row := <-rawChan
-			if row == nil {
-				break
+			row, more := <-rawChan
+			if !more {
+				close(rowChan)
```

Comment:
Should this not close `rawChan`?

Reply:
Not in this case. Since the `ReadTable` that we call above will close the `rawChan` that is the one we pass through. One the `rawChan` work is done we also need to close the `rowChan` that is passed to us here, otherwise, the program will hang waiting for this to be closed.